docker/kubernetes-tentacle/bootstrapRunner/bootstrapRunner.go

```diff
@@ -71,8 +71,7 @@ func main() {
 	Write("stdout", "Kubernetes Script Pod completed", &lineCounter)
 	Write("stdout", "##octopus[stdout-default]", &lineCounter)
 
-	//TODO: Add this back to speed things up
-	//Write("stdout", fmt.Sprintf("EOS-075CD4F0-8C76-491D-BA76-0879D35E9CFE<<>>%d", exitCode))
+	Write("stdout", fmt.Sprintf("EOS-075CD4F0-8C76-491D-BA76-0879D35E9CFE<<>>%d", exitCode), &lineCounter)
```

Comment:
could this be `debug` rather than `stdout` for consistency (as it's explicitly `debug` in the parsing code)

Reply:
Yeah good point - I didn't think of that 😁